runtime/src/non_circulating_supply.rs

```diff
@@ -79,138 +79,138 @@ pub fn calculate_non_circulating_supply(bank: &Bank) -> ScanResult<NonCirculatin
 }
 
 // Mainnet-beta accounts that should be considered non-circulating
-solana_sdk_macro::pubkeys!(
-    non_circulating_accounts,
+fn non_circulating_accounts() -> Vec<Pubkey> {
```

Comment:
The macro was generating a function with this signature. We can't use the macro as-is while ripping out `solana-sdk` since the macro implicitly brings in `solana_sdk::pubkey::Pubkey` here:
https://github.com/anza-xyz/solana-sdk/blob/1276772ee61fbd1f8a60cfec7cd553aa4f6a55f3/sdk-macro/src/lib.rs#L254-L256